project/src/store/api-action.ts

```diff
@@ -19,3 +26,97 @@ export const fetchFilmsAction = createAsyncThunk<
     return data;
   },
 );
+
+export const fetchFilmAction = createAsyncThunk<
+  Film,
+  string,
+ {
+  dispatch: AppDispatch;
+  state: State;
+  extra: AxiosInstance;
+}>(
+  'data/fetchFilm',
+  async (id, { extra: api }) => {
+    const { data } = await api.get<Film>(`${API_ROUTE.Films}/${id}`);
+    return data;
+  },
+);
+
+export const fetchSimilarFilmsAction = createAsyncThunk<
+  Film[],
+  string,
+ {
+  dispatch: AppDispatch;
+  state: State;
+  extra: AxiosInstance;
+}>(
+  'data/fetchSimilarFilm',
+  async (id, { extra: api }) => {
+    const { data } = await api.get<Film[]>(`${API_ROUTE.Films}/${id}/similar`);
+    return data;
+  },
+);
+
+export const fetchCommentsAction = createAsyncThunk<
```

Comment:
```suggestion
export const fetchComments = createAsyncThunk<
```

let's avoid using "Action" postfix everywhere

Reply:
fixed